prow/cmd/deck/static/script.js

```diff
@@ -763,6 +765,14 @@ function stateToAdj(state) {
     }
 }
 
+function gubernatorToSpyglass(gubernatorLink, buildID) {
+    var cleanLink = gubernatorLink.replace("https://k8s-gubernator.appspot.com/build/", '')
```

Comment:
Does this need to be configurable? Why do we need gubernator here?

cc @stevekuznetsov

Reply:
yeah we probably can make this first class

Reply:
Started a discussion for this in #8831